src/System.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics;
+using Xunit;
+using System;
+
+namespace System.Diagnostics.Tests
+{
+    public class EventLogSourceCreationTests : EventLogTestsBase
+    {
+        [ConditionalFact(nameof(IsProcessElevated))]
+        public void CheckSourceExistanceAndDeletion()
```

Comment:
For tests, please see my separate comments on your fork. https://github.com/Anipik/corefx/pull/1/files